frontend/src/pages/ProfilePage.jsx

```diff
@@ -18,100 +19,103 @@ function ChangeCard(props) {
     })
   }
 
   return (
-    <div>
       <Card style={{ width: '50%', padding: '10%', margin: 'auto', marginTop: '2%' }}>
-        <FormGroup controlId="email">
-          <FormLabel>Email</FormLabel>
-          <FormControl
-            value={email}
-            type="email"
-            onChange={(e) => setEmail(e.target.value)}
-          />
-        </FormGroup>
-        <FormGroup>
-          <FormLabel>First Name</FormLabel>
-          <FormControl
-            value={fname}
-            type="fname"
-            onChange={(e) => setFName(e.target.value)}
-          />
-        </FormGroup>
-        <FormGroup>
+          <FormGroup controlId="email">
+            <FormLabel>Email</FormLabel>
+            <FormControl 
+              value={email}
+              type="email"
+              onChange={(e) => setEmail(e.target.value)}
+            />
+          </FormGroup>
+          <FormGroup>
+            <FormLabel>First Name</FormLabel>
+            <FormControl
+              value={fname}
+              type="fname"
+              onChange={(e) => setFName(e.target.value)}
+            />
+          </FormGroup>
+          <FormGroup>
           <FormLabel>Last Name</FormLabel>
-          <FormControl
-            value={lname}
-            type="lname"
-            onChange={(e) => setLName(e.target.value)}
-          />
-        </FormGroup>
-        <Button type="submit" onClick={updateUserInfo}>Save</Button>
+            <FormControl
+              value={lname}
+              type="lname"
+              onChange={(e) => setLName(e.target.value)}
+            />
+          </FormGroup>
+          <Button type="submit" onClick={updateUserInfo}>Save</Button>
       </Card>
-    </div>
-
   )
 }
 
-function ProfilePage() {
-  const [user, setUser] = useState("");
-  const [userFetched, setUserFetched] = useState(false)
-  const [errorMessage, setErrorMessage] = useState("")
-  const [changeInfoCard, setChangeInfoCard] = useState(null)
+function ProfilePage(){
+    const [user, setUser] = useState("");
+    const [userFetched, setUserFetched] = useState(false)
+    const [errorMessage, setErrorMessage] = useState("")
+    const [changeInfoCard, setChangeInfoCard] = useState(null)
 
-  const id = useSelector((state) => {
-    return state.authenticationReducer.id;
-  })
+    const history = useHistory();
 
-  const genChangeCard = () => {
-    setChangeInfoCard(<ChangeCard user={user} />)
-  }
-
-  const deleteAccount = () => {
+    const id = useSelector((state) =>{
+      return state.authenticationReducer.id;
+    })
 
-  }
+    const genChangeCard = () => {
+      setChangeInfoCard(<ChangeCard user={user} />)
+    }
+
+    const deleteAccount = () => {
+      if(window.confirm("Are you sure you want to delete your account?")){
+        apiDeleteUser(id)
+        .then((response) =>{
+            console.log(response)
+        }).catch((error) => {
+          console.log(error)
+        })
+        history.push("/")
```

Comment:
ah here is the mistake i think, it'd be best to redirect back to login screen to avoid this problem https://i.gyazo.com/74b3fe2ff1bd0803fd81812110dd5967.mp4